File: OdonSys-SPA/src/app/core/models/view/invoice-detail-model.ts
export class InvoiceDetailModel {
  constructor(
    public id: string,
    public invoiceId: string,
    public procedure: string,
    public procedurePrice: number,
    public finalPrice: number,
    public dateCreated: Date,
    public userCreated: string,
    public teeth: string
  ) { }
}

File: OdonSysBackEnd/Resource/Contract/Contract.Payment/Invoices/InvoiceDetailModel.cs
namespace Contract.Payment.Invoices
{
    public record InvoiceDetailModel(
        Guid Id,
        Guid InvoiceId,
        string Procedure,
        int ProcedurePrice,
        int FinalPrice,
        DateTime DateCreated,
        string UserCreated,
        IEnumerable<string> ToothIds
    );
}
